hazelcast/src/test/java/com/hazelcast/test/starter/HazelcastStarter.java

```diff
@@ -30,20 +30,20 @@
 import java.util.concurrent.ConcurrentMap;
 
 import static com.hazelcast.test.starter.HazelcastProxyFactory.proxyObjectForStarter;
-import static com.hazelcast.test.starter.Utils.rethrow;
+import static com.hazelcast.test.starter.HazelcastStarterUtils.rethrowGuardianException;
 import static com.hazelcast.util.ExceptionUtil.rethrow;
 import static java.lang.Thread.currentThread;
 
 public class HazelcastStarter {
 
-    public static final File WORKING_DIRECTORY = Files.createTempDir();
```

Comment:
Moved to `HazelcastVersionClassloaderFuture`, same as some static methods of the parent class.